package.json

```diff
@@ -107,7 +107,7 @@
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.0.0",
-    "@department-of-veterans-affairs/generator-vets-website": "^3.3.0",
+    "@department-of-veterans-affairs/generator-vets-website": "^3.3.1",
```

Comment:
there should also be a corresponding change to `yarn.lock`. can you run `yarn` locally and see if that changes the lock file?

Reply:
Doh. Good call.